Foundation/CPKeyValueObserving.j

```diff
@@ -1300,6 +1299,10 @@ var kvoNewAndOld        = CPKeyValueObservingOptionNew | CPKeyValueObservingOpti
     if (aKeyPath === _firstPart)
```

Comment:
In what we are seeing here, is there any significant change **_before_** this line  ?